specification/machinelearning/resource-manager/Microsoft.MachineLearning/preview/2016-05-01-preview/commitmentPlans.json

```diff
@@ -387,6 +412,141 @@
     }
   },
   "definitions": {
+    "SkuListResult": {
+      "type": "object",
+      "description": "The list of commitment plan SKUs.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CatalogSku"
+          },
+          
+        }
+      }
+    },
+    "CatalogSku": {
```

Comment:
SKU API does not conform to the sku API contract as defined by ARM. The schema is not correct.

Reply:
Added missing elements per SKU API contract.